technical-doc/api.rst

```diff
@@ -5,114 +5,155 @@
 
+Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_ specification `here
+<https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
+
+.. note::
+
+   We provide an `Interactive API Specification <https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/v2&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:inputPlaceholder]=Enter%20Massa%20JSON-RPC%20server%20URL&uiSchema[appBar][ui:logoUrl]=https://massa.net/favicons/favicon.ico&uiSchema[appBar][ui:splitView]=false&uiSchema[appBar][ui:darkMode]=false&uiSchema[appBar][ui:title]=Massa&uiSchema[appBar][ui:examplesDropdown]=false&uiSchema[methods][ui:defaultExpanded]=false&uiSchema[methods][ui:methodPlugins]=true&uiSchema[params][ui:defaultExpanded]=false>`_ which documents every available method and allows you to play with the API.
+
 .. warning::
 
     Massa has a new experimental API with both Http and WebSocket support: Default port: 33036
 
     - **Http**: used for node management and blockchain interactions. e.g. http://localhost:33036
     - **WebSocket**: used for streaming blockchain events. e.g. ws://localhost:33036
 
-Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_ specification `here
-<https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
-
 Integrations
 ------------
 
-- **JavaScript**: use `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
-- **Playground**: use `Massa Playground
-  <https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/
-  v2&uiSchema[appBar][ui:input]=false&uiSchema[appBar][ui:inputPlaceholder]
-  =Enter%20Massa%20JSON-RPC%20server%20URL&uiSchema[appBar][ui:logoUrl]=
-  https://massa.net/favicons/favicon.ico&uiSchema[appBar][ui:splitView]=
-  false&uiSchema[appBar][ui:darkMode]=false&uiSchema[appBar][ui:title]=
-  Massa&uiSchema[appBar][ui:examplesDropdown]=false&uiSchema[methods]
-  [ui:defaultExpanded]=false&uiSchema[methods][ui:methodPlugins]=true&uiSchema
-  [params][ui:defaultExpanded]=false>`_.
-- **Postman**: use `Postman collection <https://github.com/massalabs/massa-docs/tree/main/postman>`_.
+**JavaScript**: see `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
+
+**Postman**: is a tool for software developers to create, test and debug API requests. You can create collections of requests in Postman workspaces for easy collaboration on projects. 
+For more information, please visit `Postman learning center <https://learning.postman.com/docs/getting-started/introduction/>`_.
+
+Find all maintained Massa Postman collections in our official `workspace <https://www.postman.com/massalabs>`_.
+
+
+Using Massa Offical Node vs Running Your Own Node on Testnet
+------------------------------------------------------------
+If you want to deploy Smart Contracts on the Massa Testnet, you have two options: using the Massa official node or running your own node.
```

Comment:
This is not specific to smart contracts. Also you don't need to use capitalised letters.

Reply:
done